frontend/styles/Timeline.module.css

```diff
@@ -0,0 +1,227 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  gap: 1em;
+  padding-bottom: 2em;
+}
+
+.labels {
+  display: flex;
+  flex-wrap: wrap;
+  user-select: none;
+  gap: 10px;
+  align-items: center;
+}
+
+.labels span {
+  text-transform: uppercase;
+  background-color: #74747d;
+  color: white;
+  border-radius: 18px;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+/* --- Timeline Card --- */
+.timelineCard {
+  cursor: pointer;
+  display: flex;
+  /* width: 1000px; */
+  width: 100%;
+  gap: 36px;
+  align-items: center;
+  border-radius: 20px;
+  /* background-color: #292a2e; */
+  background-color: #efefee;
+  padding: 20px 36px;
+  text-transform: uppercase;
+  transition: all 200ms;
+}
+
+.timelineCard:hover {
+  background-color: #404040;
+}
+
+.timelineCard .dateInfo {
+  text-align: center;
+  color: #404040;
+  transition: all 200ms;
+}
+
+.timelineCard:hover .dateInfo {
+  text-align: center;
+  color: white;
+}
+
+.timelineCard .dateInfo .day {
+  /* font-size: 4rem; */
```

Comment:
some more commented out code in this file

Reply:
removed